### DISCUSSION TITLE
美國推出AI西洋棋盤 人工智慧教你下棋

### DISCUSSION THREAD

**Discussion by 20010906** [1101-GS4719-C/Python-Community-GS4719-C#289] [General]
雖然AI可以教人如何下棋，甚至指引出正確下棋路線，但人類會不會因此而減少利用大腦思考的機會？
[AI教你下西洋棋？！](https://www-ftvnews-com-tw.cdn.ampproject.org/v/s/www.ftvnews.com.tw/news/detail/2022105W0233/amp?amp_js_v=a6&amp_gsa=1&usqp=mq331AQKKAFQArABIIACAw%3D%3D#aoh=16415152291048&referrer=https%3A%2F%2Fwww.google.com&amp_tf=%E4%BE%86%E6%BA%90%EF%BC%9A%251%24s&ampshare=https%3A%2F%2Fwww.ftvnews.com.tw%2Fnews%2Fdetail%2F2022105W0233)

**Comment by chun1203** [comment]
應該是不至於，感覺只是作為學習的另一種方式

**Comment by charlottechuang** [comment]
如果僅是用於教學方面，應該不至於讓人類腦部退化，畢竟人類還是需要去學習這個技能(?)我認為比較有可能會對於教育產業有衝擊。(教師的工作都被搶走了😥😥)

**Comment by 107602505** [comment]
我覺得人的思考模式是獨立的不會因為AI的引導就失去判斷的能力，AI的教學可以獲得知道要如何下西洋棋且彼此透過實際的對戰經驗有所成長，結果應該是互利的。

**Comment by wax75397** [comment]
AI應用於各個棋類競賽中，已行之有年了。若玩家沒有認真想過都交給AI引導，那就變成AI在玩了。AI應用於各個棋類競賽中，已行之有年了。AI可以用來研究序盤的棋譜，而到了終盤幾乎都是在比玩家的算力了。因此我認為AI依舊只是學習的工具，真的想玩的人是不會因此放棄思考的。畢竟沒有人會覺得一直機械式地移動旗子很有趣。

**Comment by zixunn** [comment]
感覺很適合提供給入門者學習，不至於導致思考能力退化，因為要精進棋藝的話始終還是要付出努力鑽研，但如果只把他當成玩具，單靠AI指引而不自己思考的話，就另當別論了。

**Comment by syt-110** [comment]
感覺很適合用來訓練推演對方或自己棋路和步數的能力

**Comment by cody910120** [comment]
努力不一定有回報但是不努力一定不會成功

**Comment by YuHengChen54** [comment]
AI通過運算來下棋真的很強，說不定這樣學各種棋類都會變超強，還記得小時候學圍棋學的超爛，早知道就跟AI學了哈哈哈

**Comment by chupochun** [comment]
但是人類有人工智慧沒有的感情，我想反而會讓人類擁有更多的動腦機會

**Comment by pekoranoinu** [comment]
AI能教人下棋，但是真正要學習下棋的是人類，人類還是需要思考。

**Comment by HuanYuuuuuu** [comment]
下棋還是要跟其他人一場一場慢慢累積經驗 發現之前從沒想過的套路才有趣

**Comment by xkllkx** [comment]
個人認為AI畢竟只是個學習工具，人跟人之間互相對奕才有趣

**Comment by bloodtrail0511** [comment]
感覺很適合初學者使用，但不能完全靠它學習，因為高手對弈時，不只有看每一手的好壞而已，還有高手之間的心理博弈

**Comment by ghost** [comment]
想太多，到最後的競演仍舊是人與人之間的對弈
